efficientdet/backbone/efficientnet_model.py

```diff
@@ -211,6 +212,17 @@ def block_args(self):
 
   def _build(self):
     """Builds block according to the arguments."""
+    # pylint: disable=g-long-lambda
+    # Keras layers have different names under graph and eager mode, so here we
+    # force their names to be the same for compatible.
+    bn_id = itertools.count(0)
+    get_bn_name = lambda: 'tpu_batch_normalization' if not next(
+        bn_id) else 'tpu_batch_normalization_' + str(next(bn_id) // 2)
+    conv_id = itertools.count(0)
+    get_conv_name = lambda: 'conv2d' if not next(
+        conv_id) else 'conv2d_' + str(next(conv_id) // 2)
+    # pylint: enable=g-long-lambda
```

Comment:
Keras does this by default in eager and graph mode.

Reply:
Yeah, this could be a potentially internal bug (or future TF feature, I am not sure, need to double confirm with TF team). We have discussed this in https://github.com/google/automl/pull/538 (It cannot pass internal tests, but it runs well in binary environment).

Since the name changes are relatively simple, we can keep this for now. If it is indeed an internal bug, then we could remove this code after fixing that bug.